README.rst

```diff
@@ -12,6 +12,11 @@ Check out the product in action: `Kubernetes-Native DevOps with Confluent Operat
 
 Get early access by registering interest here: `Confluent Operator Early Access Registration <https://events.confluent.io/confluentoperatorearlyaccess>`_
 
+This documentation repository is structured into Concepts and Scenario workflows:
```

Comment:
Could we call this something other than "documentation repository"? Such as "tutorial repository" or "example scenario repository"?